index.md

```diff
@@ -1,156 +1,20 @@
-assets/images/ # Put your images here
-assets/data/ # Put any data you want to include here (no files over 30mb)
-
-```
+![Home Image]({{ site.baseurl }}/assets/images/Wager's_Action_off_Cartagena,_28_May_1708.jpg)
 
+## Pirate Interactions on the Slave Economy 
 ---
 
-## Images, Files, and Linking
-
-*Putting in an image*
-
-```sh
-
-# Make sure to put your image in the assets/images folder (or subfolder)
-![Example Image]({{ site.baseurl }}/assets/images/example-image.png)
-
-```
-
-*Linking to Sample Data*
-
-```sh
+   The transatlantic slave trade, also known as triangle trade, was a product of the colonization and expansion of imperialistic European nations searching for new economic opportunities in the Americas. In most cases, goods would be shipped from Europe to Africa, slaves from Africa to the Americas, and goods from the Americas back to Europe, thus completing the triangle of trade. However, long slave voyages sometimes encountered problems associated with piracy. 
 
-# Make sure your data file is in the assets/data folder (or subfolder)
-[Display text]({{ site.baseurl }}/assets/data/example-data.csv)
-
-```
-
-*Linking to index.md*
-
-```sh
-
-# Make sure the page you link to is in the pages/ folder
-[Display text]({{ site.baseurl }}/index.md)
-
-```
-
-*Linking to another site page that isn't index.md*
-
-```sh
-
-# Make sure the page you link to is in the pages/ folder
-[Display text goes here]({{ site.baseurl }}/pages/put-your-page-name-here.md)
-
-```
-
-*Linking to an external page*
-
-```sh
-
-[Display text](https://www.put-your-link-url-here.html)
-
-```
+   Piracy has a long history dating back more then 2000 years, according to historian Alfred Bradford (1). Ravaging the seas in search of vast fortunes, pirates sealed their place in history as ruthlessly violent savages. One of the most popular eras in pirate history, an era of fascination among historians and the general public partially because of its place in popular culture, is known as the Golden Age (c.1700-c.1730).
 
+   One could assume based on the laws related to piracy and enacted by nations, like England and Spain during the early 1700’s, that piracy had a significant impact of the transatlantic trade. The intent of this project is to gage the economic impact that pirates had on the slave trade between 1700 and the mid 1800s, as well as to analyze how significant piracy was on the slave trade as a whole. By using data visualization software, this project will attempt to answer these questions in an effort to better understand the role of piracy in world history. 
+   
 ---
 
-## Past Project Examples
-
-* [Confederate Memorials](http://confederate-memorials-project.readthedocs.io/)
-* [The Slave Ledges](http://slave-ledger.readthedocs.io/en/latest/)
-* [An Teanga Sean: The Celtic Languages](http://an-teanga-sean-the-celtic-languages.readthedocs.io/)
+**[View the repository in GitHub](https://github.com/usf-portal/hacking-historical-texts-final)**
 
```

Comment:
Good job on this/readme.md files. For my comments and suggestions, see the README.md comments